tools/common/config.flow

```diff
@@ -133,7 +110,9 @@ getFlowDir() -> string {
 			} else if (strContains(appPath, "bin/")) {
 				takeBefore(appPath, "bin/", appPath)
 			} else {
-				flowDirByFlowc();
+				println("WARNING: Could not determine the flow directory.");
```

Comment:
we have $FLOW environment variable set to the flow9 folder, why don't we use it here?